src/tools/devtools/inspector.md

```diff
@@ -517,6 +517,42 @@ This feature prevents otherwise-identical `const` Widgets from
 being considered equal in debug builds. For more details, see
 the discussion on [common problems when debugging][].
 
+## Inspector Settings
+![The Flutter Inspector Settings dialog]({{site.url}}/assets/images/docs/tools/devtools/flutter_inspector_settings.png){:width="100%"}
+
+### Enable Hover Inspection
+
+Hovering over any widget displays its properties and values.
+
+Toggling this value, will enable or disable the hover inspection functionality.
+
+### Package Directories
+
+By default DevTools limits the visible Widgets in the widget tree, to those from the project's root directory and those from Flutter. 
```

Comment:
```suggestion
By default, DevTools limits the widgets displayed in the widget tree
to those from the project's root directory, and those from Flutter. 
```

Reply:
I would add a sentence after this: "This filtering only applies to the widgets in the Inspector Widget Tree (left side of the Inspector) -- not the Widget Details Tree (right side of the Inspector in the same tab view as the Layout Explorer). In the Widget Details Tree, you will be able to see all widgets in the tree from all packages."